src/rules/property-no-unknown/README.md

```diff
@@ -85,3 +87,20 @@ The following patterns are *not* considered warnings:
   custom: 10px;
 }
 ```
+### `checkPrefixed: true | false` (default: `false`)
+
+This option forces to check vendor prefixed properties.
```

Comment:
What is the best way to explain why this is no enabled by default? I may just say "For compatibility with `csslint` this option is not enabled by default". Or other way is to copy/past from csslint docs - "All vendor-prefixed properties (those beginning with a -) are ignored because vendors may add in their own properties at any point in time, and there are no canonical lists of these properties. This behavior is different than a CSS validator, which warns when a vendor-prefixed property is used."